frontend/i18n.js

```diff
@@ -0,0 +1,71 @@
+import i18n from 'i18next';
```

Comment:
This should probably go under `frontend/public` if it's part of the console app.

Reply:
Should other config files and the conversion js go there too?

Reply:
> Should other config files and the conversion js go there too?

No, not if they're not part of the console app itself.